flask/README.rst

```diff
@@ -48,6 +48,8 @@ Create database schema:
     $ cd YOUR_PROJECT_DIR/flask/
     $ python db.py
 
+Although you shouldn't need to do it since the application will do that in a idempotent way.
```

Comment:
I think we can remove `Create database schema` subsection then.

Reply:
Done!